### PR TITLE
NXP: Add watchdog support for the KE1xF SoC

### PR DESCRIPTION
**Pull request by henrikbrixandersen**
This patch series adds support for the WDOG32 watchdog module present in the NXP Kinetis KE1xF SoC series and documents it in the NXP TWR-KE18F development board documentation.

### CODE REVIEW

File: drivers/watchdog/wdt_mcux_wdog32.c [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+ * Copyright (2) 2019 Vestas Wind Systems A/S
+ *
+ * Based on wdt_mcux_wdog.c, which is:
+ * Copyright (c) 2018, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <watchdog.h>
+#include <clock_control.h>
```
Comment (MaureenHelm) [inline drivers/watchdog/wdt_mcux_wdog32.c]:
```
#include <drivers/watchdog.h>
#include <drivers/clock_control.h>
```

**Review by MaureenHelm — CHANGES_REQUESTED** [review]
Just one nit, otherwise looks good

File: drivers/watchdog/wdt_mcux_wdog32.c (hunk 1 above)
Comment (henrikbrixandersen) [reply drivers/watchdog/wdt_mcux_wdog32.c]: Thanks, fixed.

**Review by MaureenHelm — APPROVED** [review]

**Review by dbkinder — APPROVED** [review]
doc change LGTM

**Pull request merged**